brave/browser-laptop
# Split scroll between titles and folders for BM manager

- Auditors: @bsclifton
- Close #5076

Test Plan:
- Have a lot of bookmarks
- Go to about:bookmarks
- Ensure that:
  - Layout is fixed
  - You can scroll between folders (left-hand side column) and titles (right-hand side column) wouldn't scroll together
  - Less-than-screen-height number of bookmakrs shouldn't let you scroll (layout is fixed)
  - At small sizes we fallback to what we have and you can scroll gracefully
